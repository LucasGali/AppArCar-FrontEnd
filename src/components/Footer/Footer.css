.footer {
  position: relative;
  height: 89px;
  left: 0;
  bottom: -10vh;
  width: 100%;
  background-color: black;
  color: white;
  text-align: center;
  z-index: 2;
}

.footer-links-wrapper {
  display: flex;
  list-style: none;
  justify-content: space-evenly;
}

.footer-links-wrapper div:first-child {
  display: flex;
  flex-direction: row;
  align-self: center;
  font-size: 16px;
}

.first-set {
  display: flex;
  flex-direction: column;
}

.second-set {
  display: flex;
  flex-direction: column;
}

.footer-links-wrapper div:first-child a {
  margin-left: 5px;
}

.footer-links-wrapper div:first-child a:hover {
  color: bisque;
  cursor: pointer;
}

.footer-links-wrapper div li {
  margin: 10px;
}

.footer-links-wrapper div a {
  display: flex;
  color: white;
  text-decoration: none;
}

.footer-links-wrapper div a span {
  margin-left: 5px;
}

.footer-links-wrapper div a span:hover {
  color: bisque;
  cursor: pointer;
}

.footer-links-wrapper dl,
ol,
ul {
  margin: 0;
}

@media only screen and (max-width: 1000px) {
  .footer {
    height: 15vh;
  }
  .footer-links-wrapper div:first-child {
    display: none;
  }

  .first-set {
    margin: 12px 20px 0px;
    font-size: 15px;
    padding: 10px;
  }

  .second-set {
    margin: 12px 20px 0px;
    font-size: 15px;
    padding: 10px;
  }
}

@media only screen and (max-height: 900px) {
  .footer {
    height: 15vh;
  }
  .footer-links-wrapper div:first-child {
    display: none;
  }

  .first-set {
    margin: 12px 20px 0px;
    font-size: 15px;
    padding: 10px;
  }

  .second-set {
    margin: 12px 20px 0px;
    font-size: 15px;
    padding: 10px;
  }
}

@media only screen and (max-height: 700px) {
  .footer {
    height: 20vh;
  }
  .footer-links-wrapper div:first-child {
    display: none;
  }

  .first-set {
    margin: 12px 20px 0px;
    font-size: 15px;
    padding: 10px;
  }

  .second-set {
    margin: 12px 20px 0px;
    font-size: 15px;
    padding: 10px;
  }
}
