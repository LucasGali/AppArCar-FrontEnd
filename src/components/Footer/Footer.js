import React from 'react'
import iconSet from 'icons.json';
import IcomoonReact from 'icomoon-react';
import text from 'utils/text';
import './Footer.css'

const Footer = () => {
  return (
    <div className="footer">
      <ul className="footer-links-wrapper">
        <div>
          <p>{text.footer.unsta}</p>
          <a href="https://apparcar-unsta.netlify.app/">{text.footer.site}</a>
        </div>
        <div className="first-set">
          <li className="home-unsta">
            <a href="https://www.unsta.edu.ar/ingenieria/desarrollo-y-calidad-de-software/">
              <IcomoonReact
                className=""
                iconSet={iconSet}
                color="white"
                size={20}
                icon="home"
              />
              <span>{text.footer.unstaBtn}</span>
            </a>
          </li>
          <li className="stats">
            <a href="https://drive.google.com/drive/folders/10A62Pe56qRWPHoFYLAdy1iYGZcl4M08d?usp=sharing">
              <IcomoonReact
                className=""
                iconSet={iconSet}
                color="white"
                size={20}
                icon="file-zip"
              />
              <span>{text.footer.gdocBtn}</span>
            </a>
          </li>
        </div>
        <div className="second-set">
          <li className="board">
            <a href="https://trello.com/proyectofinalunstatdcs">
              <IcomoonReact
                className=""
                iconSet={iconSet}
                color="white"
                size={20}
                icon="map"
              />
              <span>{text.footer.trelloBtn}</span>
            </a>
          </li>
          <li className="repo">
            <a href="https://github.com/ejrico/apparcar">
              <IcomoonReact
                className=""
                iconSet={iconSet}
                color="white"
                size={20}
                icon="github"
              />
              <span>{text.footer.repoBtn}</span>
            </a>
          </li>
        </div>
      </ul>
    </div>
  );
}

export default Footer
