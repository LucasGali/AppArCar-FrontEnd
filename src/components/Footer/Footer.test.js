import { render, screen } from '@testing-library/react';
import Footer from './Footer';
import { MemoryRouter } from 'react-router-dom';

describe('Footer', () => {
  it('renders default form according to snapshot', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <Footer />
      </MemoryRouter>,
    );
    expect(asFragment(<Footer />)).toMatchSnapshot();
  });
});
