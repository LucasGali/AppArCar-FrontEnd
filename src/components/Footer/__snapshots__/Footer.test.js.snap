// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders default form according to snapshot 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <ul
      class="footer-links-wrapper"
    >
      <div>
        <p>
          Universidad del Norte Santo Tomás de Aquíno - Tucumán, Argentina, 2021 |
        </p>
        <a
          href="https://apparcar-unsta.netlify.app/"
        >
          apparcar-unsta.netlify.app
        </a>
      </div>
      <div
        class="first-set"
      >
        <li
          class="home-unsta"
        >
          <a
            href="https://www.unsta.edu.ar/ingenieria/desarrollo-y-calidad-de-software/"
          >
            <svg
              class=""
              height="20"
              style="display: inline-block; vertical-align: middle;"
              viewBox="0 0 1024 1024"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M1024 590.444l-512-397.426-512 397.428v-162.038l512-397.426 512 397.428zM896 576v384h-256v-256h-256v256h-256v-384l384-288z"
                style="fill: white;"
              />
            </svg>
            <span>
              - UNSTA - TDSC
            </span>
          </a>
        </li>
        <li
          class="stats"
        >
          <a
            href="https://drive.google.com/drive/folders/10A62Pe56qRWPHoFYLAdy1iYGZcl4M08d?usp=sharing"
          >
            <svg
              class=""
              height="20"
              style="display: inline-block; vertical-align: middle;"
              viewBox="0 0 1024 1024"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M917.806 229.076c-22.208-30.292-53.174-65.7-87.178-99.704s-69.412-64.964-99.704-87.178c-51.574-37.82-76.592-42.194-90.924-42.194h-496c-44.112 0-80 35.888-80 80v864c0 44.112 35.884 80 80 80h736c44.112 0 80-35.888 80-80v-624c0-14.332-4.372-39.35-42.194-90.924v0 0zM785.374 174.626c30.7 30.7 54.8 58.398 72.58 81.374h-153.954v-153.946c22.98 17.78 50.678 41.878 81.374 72.572v0 0zM896 944c0 8.672-7.328 16-16 16h-736c-8.672 0-16-7.328-16-16v-864c0-8.672 7.328-16 16-16 0 0 495.956-0.002 496 0v224c0 17.672 14.322 32 32 32h224v624z"
                style="fill: white;"
              />
              <path
                d="M256 64h128v64h-128v-64z"
                style="fill: white;"
              />
              <path
                d="M384 128h128v64h-128v-64z"
                style="fill: white;"
              />
              <path
                d="M256 192h128v64h-128v-64z"
                style="fill: white;"
              />
              <path
                d="M384 256h128v64h-128v-64z"
                style="fill: white;"
              />
              <path
                d="M256 320h128v64h-128v-64z"
                style="fill: white;"
              />
              <path
                d="M384 384h128v64h-128v-64z"
                style="fill: white;"
              />
              <path
                d="M256 448h128v64h-128v-64z"
                style="fill: white;"
              />
              <path
                d="M384 512h128v64h-128v-64z"
                style="fill: white;"
              />
              <path
                d="M256 848c0 26.4 21.6 48 48 48h160c26.4 0 48-21.6 48-48v-160c0-26.4-21.6-48-48-48h-80v-64h-128v272zM448 768v64h-128v-64h128z"
                style="fill: white;"
              />
            </svg>
            <span>
              - G-Doc
            </span>
          </a>
        </li>
      </div>
      <div
        class="second-set"
      >
        <li
          class="board"
        >
          <a
            href="https://trello.com/proyectofinalunstatdcs"
          >
            <svg
              class=""
              height="20"
              style="display: inline-block; vertical-align: middle;"
              viewBox="0 0 1024 1024"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 192l320-128v768l-320 128z"
                style="fill: white;"
              />
              <path
                d="M384 32l320 192v736l-320-160z"
                style="fill: white;"
              />
              <path
                d="M768 224l256-192v768l-256 192z"
                style="fill: white;"
              />
            </svg>
            <span>
              - Trello Board
            </span>
          </a>
        </li>
        <li
          class="repo"
        >
          <a
            href="https://github.com/ejrico/apparcar"
          >
            <svg
              class=""
              height="20"
              style="display: inline-block; vertical-align: middle;"
              viewBox="0 0 1024 1024"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M512.008 12.642c-282.738 0-512.008 229.218-512.008 511.998 0 226.214 146.704 418.132 350.136 485.836 25.586 4.738 34.992-11.11 34.992-24.632 0-12.204-0.48-52.542-0.696-95.324-142.448 30.976-172.504-60.41-172.504-60.41-23.282-59.176-56.848-74.916-56.848-74.916-46.452-31.778 3.51-31.124 3.51-31.124 51.4 3.61 78.476 52.766 78.476 52.766 45.672 78.27 119.776 55.64 149.004 42.558 4.588-33.086 17.852-55.68 32.506-68.464-113.73-12.942-233.276-56.85-233.276-253.032 0-55.898 20.004-101.574 52.76-137.428-5.316-12.9-22.854-64.972 4.952-135.5 0 0 43.006-13.752 140.84 52.49 40.836-11.348 84.636-17.036 128.154-17.234 43.502 0.198 87.336 5.886 128.256 17.234 97.734-66.244 140.656-52.49 140.656-52.49 27.872 70.528 10.35 122.6 5.036 135.5 32.82 35.856 52.694 81.532 52.694 137.428 0 196.654-119.778 239.95-233.79 252.624 18.364 15.89 34.724 47.046 34.724 94.812 0 68.508-0.596 123.644-0.596 140.508 0 13.628 9.222 29.594 35.172 24.566 203.322-67.776 349.842-259.626 349.842-485.768 0-282.78-229.234-511.998-511.992-511.998z"
                style="fill: white;"
              />
            </svg>
            <span>
              - Repositorio
            </span>
          </a>
        </li>
      </div>
    </ul>
  </div>
</DocumentFragment>
`;
